nvOC_miner_update.sh

```diff
@@ -12,53 +12,24 @@ then
   CUDA_VER="8.0"
 fi
 
-## Miner versions and tarballs
-
-ANXccminer_ver_8="1.0"
-ANXccminer_compiled_tarball_ver_8="ANXccminer.tar.xz"
-ANXccminer_src_hash_ver_8="cd6fab68823e247bb84dd1fa0448d5f75ec4917d"
-
-ASccminer_ver_8="1.0"
-ASccminer_compiled_tarball_ver_8="ASccminer.tar.xz"
-ASccminer_src_hash_ver_8="baf6c9e4e36c9cc1b67698ee2977d445f598c573"
-
-KTccminer_ver_8="8.20"
-KTccminer_compiled_tarball_ver_8="KTccminer.tar.xz"
-KTccminer_src_hash_ver_8="c5ab73837c8024f1e6b8fe7ad46e6881fb8366e6"
-
-KTccminer_ver_9="8.22"
-KTccminer_compiled_tarball_ver_9="KTccminer-8.22.tar.xz"
-KTccminer_src_hash_ver_9="2e457923f3125fbaedf5d8ba1f7d0fafc85b0ba8"
-
-KXccminer_ver_8="0.1"
-KXccminer_compiled_tarball_ver_8="KXccminer.tar.xz"
-KXccminer_src_hash_ver_8="7d41d49b92db27b9ab80270adaa92f6b06d1ef78"
-
-LOLMINER_ver="0.43"
-LOLMINER_compiled_tarball="lolMiner_v043_Lin64.tar.xz"
+# Installing Energiminer Dependencies"
+if [ ! -f /etc/apt/sources.list.d/ubuntu-toolchain-r-ubuntu-test-bionic.list ] 
+then 
+  echo "Installing Energiminer Dependencies"
+  sudo apt --assume-yes install software-properties-common 
+  sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+  sudo apt --assume-yes install gcc-4.9 
+  sudo apt upgrade libstdc++6 
+fi
 
-MSFTccminer_ver_8="2.2.5"
-MSFTccminer_compiled_tarball_ver_8="MSFTccminer.tar.xz"
-MSFTccminer_src_hash_ver_8="78dad7dd659eae72a07d2448de62b1946c1f2b41"
+## Miner versions and tarballs
 
-NAccminer_ver_8="2.2"
-NAccminer_compiled_tarball_ver_8="nanashi-ccminer-2.2-mod-r2.tar.xz"
-NAccminer_src_hash_ver_8="8affcb9cd09edd917d33c1ed450f23400f571bdb"
+LOLMINER_ver="0.6"
+LOLMINER_compiled_tarball="lolMiner_v06_Lin64.tar.xz"
```

Comment:
The above two lines are useless, we need such variables only for legacy builtin miners. Nothing urgent btw.